WalletWasabi/WabiSabi/Client/CredentialDependencies/DependencyGraphExtensions.cs

```diff
@@ -1,8 +1,7 @@
 using System;
 using System.Linq;
-using WalletWasabi.WabiSabi.Client.CredentialDependencies;
 
-namespace WalletWasabi.Tests.Helpers
+namespace WalletWasabi.WabiSabi.Client.CredentialDependencies
```

Comment:
not sure about this renaming, @lontivero previously suggested to keep it in the test project, but if we want to spit it out of `CoinJoinClient` that no longer works. is that enough of a justification?

Reply:
It is a good idea, but I don't know this it seemed like not trivial to me. Can you do it in a follow-up PR?

Reply:
This is done.